src/Arcus.EventGrid.Tests.Integration/Publishing/EventPublishingTests.cs

```diff
@@ -107,6 +107,37 @@ await EventGridPublisherBuilder
             AssertReceivedEvent(secondEventId, secondEvent.EventType, eventSubject, licensePlate, secondReceivedEvent);
         }
 
+        [Fact]
+        public async Task PublishMultipleEvents_WithBuilder_ValidParameters_SucceedsWithTimeout()
+        {
+            // Arrange
+            var topicEndpoint = Configuration.GetValue<string>("Arcus:EventGrid:TopicEndpoint");
+            var endpointKey = Configuration.GetValue<string>("Arcus:EventGrid:EndpointKey");
+            var events = 
```

Comment:
Best to align with existing tests to keep it consistent.

Either change those as well or change this one.

Reply:
Ok, done!

Reply:
Please don't use abbreviations, I presume `f` is the guid or?

Reply:
Yeah, about that. It's a bit of a habbit in F# to use `f`, `g`, `h`, ... as names for functions and `x`, `y`, `z`, ... for names of values. But yeah; was purely a habbit that I used `f`.

It's a function that creates a `Guid` so we have unique `Guid`'s for every event.

Reply:
Renamed to `newGuid`